src/NServiceBus.Testing/Invocations.cs

```diff
@@ -199,6 +199,11 @@ public ExpectedNotDeferMessageInvocation()
 
     class ExpectedReplyToOriginatorInvocation<M> : ExpectedInvocation<ReplyToOriginatorInvocation<M>>
     {
+        public ExpectedReplyToOriginatorInvocation(bool negate = false)
```

Comment:
The only thing I would nitpick about is this one. It seems that for the majority of Expected classes we have a NonNegated and a Negated version. The negated version inherits from the NonNegated one and sets `Negated = true`. This one is alien to that design by allowing to pass in the `negate` flag. // @timbussmann